### DISCUSSION TITLE
Docker-compose & Dokku deployment

### DISCUSSION THREAD

**Discussion by 53RG1005** [dokku/dokku#5546] [Q&A]
I create an application where I have my API created on node (+ mariadb) with socket and PHP application which work with this API. The API based on api.domain.com and application on domain.com. Before I deploy this with docker-compose and traefik.

Also, I had access to uploaded files folder for backup. How to create this volume with dokku and have access to this forled for backup?

Please, can someone explain me, how to deploy it with dokku?

**Comment by josegonzalez** [comment]
Have you checked out our getting started documentation? Specifically, after you install Dokku, you get a link to this tutorial: https://dokku.com/docs/deployment/application-deployment/
